include/LightGBM/config.h

```diff
@@ -333,7 +333,7 @@ struct Config {
   // alias = subsample_freq
   // desc = frequency for bagging
   // desc = ``0`` means disable bagging; ``k`` means perform bagging at every ``k`` iteration. Every ``k``-th iteration, LightGBM will randomly select ``bagging_fraction * 100 %`` of the data to use for the next ``k`` iterations
-  // desc = **Note**: to enable bagging, ``bagging_fraction`` should be set to value smaller than ``1.0`` as well
+  // desc = **Note**: to enable bagging, ``bagging_fraction`` should be set to a value between ``0.0`` and ``1.0``
```

Comment:
Should we maybe change the wording a bit? The original intent is to clarify that `bagging_fraction=1.0` disables bagging. Between usually means inclusive on both sides, but this param should be greater than 0 and less than 1 to enable it.

Reply:
> Should we maybe change the wording a bit? The original intent is to clarify that `bagging_fraction=1.0` disables bagging. Between usually means inclusive on both sides, but this param should be greater than 0 and less than 1 to enable it.

Clarification is made

Reply:
sure